dbt/include/athena/macros/utils/timestamps.sql

```diff
@@ -3,7 +3,18 @@
 %}
 
 {% macro athena__current_timestamp() -%}
-    cast(now() as timestamp(6))
+  {{ cast_timestamp('now()') }}
+{%- endmacro %}
+
+
+{% macro cast_timestamp(timestamp_col) -%}
+  {%- set config = model['config'] -%}
```

Comment:
smart idea. 

What if we are using this in a context without config?

Reply:
Good point, do you know if the `model` variable is always available? If so, I can change it to 
```
{%- set config = model.get('config', {}) -%}
```

Reply:
you can try, I believe that for tests is not available.

Reply:
Ah right, you meant dbt tests of course. I checked that, and it works as expected:

```
SELECT {{ cast_timestamp('now()') }}
```
compiles to 
```
SELECT cast(now() as timestamp)
```

And 
```
{{ config(table_type='iceberg') }}

SELECT {{ cast_timestamp('now()') }}
```
compiles to
```
SELECT cast(now() as timestamp(6))
```

This was done by adding the mentioned content to `tests/test_timestamp.py` and then running `dbt test --select test_timestamp`

Reply:
So it works with and without config. By default it will use the glue version, which makes sense imo

Reply:
Yes absolutely, thanks for checking